static/app/components/codeSnippet.tsx

```diff
@@ -0,0 +1,104 @@
+import 'prism-sentry/index.css';
+
+import {Fragment, useEffect, useRef, useState} from 'react';
+import styled from '@emotion/styled';
+import Prism from 'prismjs';
+
+import Tooltip from 'sentry/components/tooltip';
+import {IconCopy} from 'sentry/icons';
+import {t} from 'sentry/locale';
+
+const PreContainer = styled('pre')<{unsetBorderRadiusTop?: boolean}>`
+  overflow-x: scroll;
+  ${p =>
+    p.unsetBorderRadiusTop
+      ? `
+  border-top-left-radius: 0px;
+  border-top-right-radius: 0px;
+  `
+      : null}
+
+  word-break: break-all;
+  white-space: pre-wrap;
+
+  code {
+    white-space: pre;
+  }
+`;
+const UnstyledButton = styled('button')`
+  all: unset;
+  cursor: pointer;
+`;
+
+const CodeContainerActionBar = styled('div')`
+  display: flex;
+  justify-content: end;
+  gap: 10px;
+  padding: 10px;
+
+  color: ${p => p.theme.white};
+  font-size: ${p => p.theme.fontSizeSmall};
+  border-bottom: 1px solid ${p => p.theme.purple200};
+  background: #251f3d;
```

Comment:
And one of the predefined colors in `theme.tsx`?

Reply:
this color isn't in theme, at least not that i'm aware of. the background color of  the `<pre/>` is globally styled so i copied it, but 💡.... 

i can porbably just make this element a `pre` aswell and let the globals take over